gatsby/create-pages/create-blog-posts.js

```diff
@@ -31,7 +32,9 @@ const createBlogPosts = async ({ actions, reporter, graphql }) => {
     createPage({
       path: appendTrailingSlash(node.frontmatter.path),
       component: BLOG_POST_TEMPLATE_PATH,
-      context: {},
+      context: {
+        publicationDate: node.frontmatter.date,
```

Comment:
That's only fore the sitemap. For a brief moment I thought, why should I pass in the data as the page itself queries for all of that 👍